flink-runtime/src/main/java/org/apache/flink/runtime/jobmanager/JobManagerOptions.java

```diff
@@ -32,6 +32,10 @@
 	public static final ConfigOption<Integer> MAX_ATTEMPTS_HISTORY_SIZE =
 			key("job-manager.max-attempts-history-size").defaultValue(16);
 
+	public static final ConfigOption<String> ARCHIVE_DIR =
+			key("job-manager.archive.dir")
```

Comment:
I know that you are following this classes example, but we have `jobmanager` as the regular config option spelling in the shipped default config. We need to change this and create a issue to rename the max attempts key accordingly.